subsys/usb/device_next/class/usbd_cdc_ncm_internal.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2024 Hardy Griech
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#ifndef __USBD_CDC_NCM_LOCAL_H_
+#define __USBD_CDC_NCM_LOCAL_H_
+
+
+#ifndef CFG_CDC_NCM_ALIGNMENT
+#define CFG_CDC_NCM_ALIGNMENT                 4
+#endif
+#if (CFG_CDC_NCM_ALIGNMENT != 4)
+/*  headers and start of datagrams in structs have to be aligned differently */
+#error "CFG_CDC_NCM_ALIGNMENT must be 4"
+#endif
```

Comment:
Is it not the same as just `#define CFG_CDC_NCM_ALIGNMENT                 4`?
Please remove all the `CFG_` prefixes and move whatever make sense to Kconfig options.

Reply:
I have "reserved" those CFGs for future extensions.  Currently there is no meaning in them, so they don't appear in Kconfig.  Will change it.